pkgs/development/python-modules/pyspread/default.nix

```diff
@@ -25,32 +18,46 @@ buildPythonPackage rec {
     sha256 = "117268cbeb225e0aca4c59381b7bdf5333bf95ca85c1db410a326a887a74f3da";
   };
 
-  propagatedBuildInputs = [ numpy wxPython matplotlib pycairo python-gnupg xlrd xlwt jedi pyenchant basemap pygtk ];
-  # Could also (optionally) add pyrsvg and python bindings for libvlc
+  requiredPythonDependencies = [
```

Comment:
`buildPythonPackage` does not have these parameters, so move them into a let..in expression